data/rail_network/rail_interchange_impedances.yaml

```diff
@@ -0,0 +1,29 @@
+# Data/rules for assigning impedances at rail interchanges
+
+# Defaults for interchanges not named in the rule 260 section
+class1_to_class1_default_impedance: 1000
+class1_to_shortline_default_impedance: 350
+
+# Defaults by Rule 260 interchange
+impedance_by_rule_260_interchange: {
```

Comment:
So I do think we should provide a structure here, and I might suggest that the natural choice might be `impedances`, which should contain a list of objects:

```
impedances:
  - jr260: "BATON" 
    latitude: 30.1234
    longitude: -75.088
  - jr260: "SOMETHING_ELSE" 
    latitude: 30.1234
    longitude: -75.088

```